site/faq.md

```diff
@@ -101,21 +101,29 @@ system in order to read and update the manifests.
 
 ### How do I give Flux access to an image registry?
 
-Flux transparently looks at the image pull secret that you give for a
-workload, and thereby uses the same credentials that Kubernetes uses
-for pulling each image. If your pods are running, Kubernetes has
-pulled the images, and Flux should be able to access them.
+Flux transparently looks at the image pull secrets that you to
+workloads and service accounts, and thereby uses the same credentials
```

Comment:
The first half of the sentence is a bit off. Can you rephrase to make it clearer?

Reply:
owps! corrected.